scripts/deploy-nomad-web-release.sh

```diff
@@ -371,6 +371,11 @@ if [[ "$JAAS_TOKEN_URL" != "null" ]]; then
     export NOMAD_VAR_api_jaas_token_url="$JAAS_TOKEN_URL"
 fi
 
+JAAS_CONFERENCE_CREATOR_URL="$(cat $ENVIRONMENT_CONFIGURATION_FILE | yq eval .jitsi_meet_api_jaas_conference_creator_url -)"
```

Comment:
So this part isn't needed so long as the variable is named jitsi_meet_api_jaas_conference_creator_url in all the places.  So if you rename your variable in the nomad.hcl to jitsi_meet_api_jaas_conference_creator_url then it should be imported automatically via the magic in this line.

`eval $(yq '.. | select(tag == "!!int" or tag == "!!str" or tag == "!!bool") |  "export NOMAD_VAR_"+(path | join("_")) + "=\"" + . + "\""' $ENVIRONMENT_CONFIGURATION_FILE)`